src/main/scala/mesosphere/marathon/state/Instance.scala

```diff
@@ -20,7 +20,8 @@ case class Instance(
     state: InstanceState,
     tasksMap: Map[Task.Id, Task],
     runSpecVersion: Timestamp,
-    reservation: Option[Reservation]) extends MarathonState[Protos.Json, Instance] {
+    reservation: Option[Reservation],
+    role: String) extends MarathonState[Protos.Json, Instance] {
```

Comment:
This should be `Option[String]` as it is the storrge model.

Reply:
Why though? It might make the migration a little bit easier, but we'd have to take the optionality into account when converting from storage state to core conversion.

Reply:
Changed it.